ci-overview.md

```diff
@@ -26,30 +26,37 @@ The results of every job in the CI pipeline are summarized on the bottom of the
 
 ### Special CI Commands
 
-Special commands are provided that can be included in commit messages to direct the CI pipeline execution:
+Special commands can be included in the most recent commit message to control which jobs are spawned for the next pull-request CI run.
+These commands can be combined with the [override matrix](#temporarily-overriding-the-pull-request-matrix) for even more fine-grained control.
 
-- `[skip ci]`: Skips the entire CI pipeline. Useful for documentation changes or others that don't require CI validation.
+- `[skip-<component>]`: Skips a subset of the CI jobs. These commands will block the PR from being merged while present in the last commit message of the branch. Recognized components are:
+  - `[skip-matrix]`: Skip all build and test jobs specified in `ci/matrix.yaml`.
+  - `[skip-vdc]`: Skip all "Validate Devcontainer" jobs.
+  - `[skip=docs]`: Skip the documentation verification build.
```

Comment:
I assume this one isn't different than the rest? 
```suggestion
  - `[skip-docs]`: Skip the documentation verification build.
```

Reply:
D'oh, yep that's a typo